web-test-runner.utils.js

```diff
@@ -277,4 +257,26 @@ export const configuredVisualRegressionPlugin = () =>
                 ...nameParts
             );
         },
+        failureThresholdType: 'percent',
+        failureThreshold: 3,
     });
+
+export const filterBrowserLogs = (log) => {
```

Comment:
created a browser log filter that we can add/improve overtime to clean up the noise in our outputs so we can see important logs when they bubble up